packages/preview/isec-master-thesis-template/1.0.0/src/lib.typ

```diff
@@ -78,7 +79,7 @@
 	// Set document metadata
 	#set document(
 		title: title,
-		author: author.at(0).text,
+		author: if type(author) == "content" { author.text } else { author.at(0).text },
```

Comment:
string/type comparison will be removed in Typst 0.13, you should use `content` and not `"content"`.